presto-docs/src/main/sphinx/admin/web-interface.rst

```diff
@@ -3,9 +3,20 @@ Web UI
 ======
 
 Presto provides a web-based user interface (UI) for monitoring a Presto cluster
-and managing queries. The web UI is accessible on the Presto coordinator via
+and managing queries. The Web UI is accessible on the coordinator via
 HTTP/HTTPS, using the corresponding port number specified in the coordinator
-:ref:`config_properties`.
+:ref:`config_properties`. If desired, the Web UI can be disabled with
+``web-ui.enabled=false``.
+
+The Web UI requires users to log in on the initial page. A default Presto
+installation has no authorization and authentication enabled and you can provide
+any username to log in successfully. Best practice is to use the same username,
+when connecting to other systems in your organization, including running queries
+on Presto with the CLI or some other application.
+
+With username/password authentication enabled, you are required to configure the
```

Comment:
I would maybe put that the other way around.... When the server is configured to use HTTPS/TLS, a username/password authenticator, such as LDAP or FILE, must be configured.

Reply:
Hm .. thats kinda different from what I understand to be the case .. e.g. i thought you could have https/tls but no authentication just like the default behaviour? On the other hand if you have authenticator .. you need to use https/tls since this would be completely insecure otherwise.

Reply:
Can you confirm which is correct?

Reply:
I changed to your suggestion for now but we should still confirm this to be true .. and if so.. I would love to understand the reasoning.

Reply:
Reasoning came up in slack. I am good with this.